packages/assets/assets/LongLeftArrow.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg width="100%" height="100%" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
     <path d="M9.57 5.92969L3.5 11.9997L9.57 18.0697" stroke="#F2F2F2" stroke-width="1.5"
```

Comment:
The image looks white when previewed using GitHub/IDE. Let's update the stroke to `#0d0d0d`

```suggestion
    <path d="M9.57 5.92969L3.5 11.9997L9.57 18.0697" stroke="#0d0d0d" stroke-width="1.5"
```